package.json

```diff
@@ -1,51 +1,56 @@
 {
-  "name": "duplicate-package-checker-webpack-plugin",
-  "version": "2.1.0",
-  "description":
-    "Webpack plugin that warns you when multiple versions of the same package exist in a build.",
-  "main": "./lib/index.js",
-  "scripts": {
-    "test": "jest test/**/*.test.js --runInBand",
-    "build": "babel src --out-dir lib",
-    "prepublish": "npm run build",
-    "precommit": "lint-staged",
-    "pretty": "prettier --write \"src/**/*.{json,js}\" \"test/**/*.{json,js}\""
+  "name": "@cerner/duplicate-package-checker-webpack-plugin",
```

Comment:
Are we starting to create scoped npm packages?

Reply:
We are with this package. Going forward, I would prefer scoped packages.

Reply:
We are doing so here due to this being a fork of another open source project. This allows us to keep the previous projects commit history and name while packaging/releasing it separately.